### ISSUE TITLE
[ACTIVE_ARBITRATED] SWARM-1275 — PR1100 exact-head 1581dc99 independent qualification

### ISSUE THREAD

**Issue by Oleksii-debug** [Oleksii-debug/12-6-ai.#1275]
SWARM_PROTOCOL: SWARM-300-V2
SWARM_CONTROL: #723
SWARM_WORKER_ID: SWARM-1275
SWARM_LANE_KEY: AUDIT-B|PR1100|UBUNTU-IRC-REAL-EXECUTION-EVIDENCE|EXACTHEAD-1581DC99-V2
PREFERRED_LANE: #2
PREFERRED_WORK_KIND: DATA_SOURCE_OR_PIPELINE
SELECTED_WORK_KIND: INDEPENDENT_VERIFY / REAL_EXECUTION_EVIDENCE / CONVERGENCE
PARENT_ISSUES: #4 #548 #723 #769 #913 #1084 #1096 #1100 #1115 #1238 #1272
STATUS: ACTIVE_ARBITRATED
BASE_SHA: 7accc766d76693a69dabb5ff853cb35040ec4728
TARGET_PR: #1100
TARGET_HEAD: 1581dc99a87a38eda777988578ba0a8a9af689d8
REJECTED_HEAD: ab457e641d3ca5252cdbbd7fd5e2928d3c761521
PRODUCER_REPAIR_OWNER: #1272 (RELEASED / READY_FOR_FRESH_INDEPENDENT_AUDIT)
CURRENT_INTEGRATION_OWNER: #1238
EXACT_HEAD_SHARED_CI: 34604427566=SUCCESS
CLAIMED_AT_UTC: 2026-09-11T13:31:00Z
LEASE_UNTIL_UTC: 2026-09-11T19:31:00Z

POSTCLAIM DIRECT OPEN ARBITRATION: WIN
- fetched direct GitHub OPEN issue collection with per_page=100, created-desc pages 1..8;
- page 9 is empty, proving the current OPEN collection was exhausted;
- exact repaired head 1581dc99a87a38eda777988578ba0a8a9af689d8 has zero matches on direct pages 2..8;
- page 1 contains this #1275 claim and released producer #1272 requesting a NEW different-worker audit; #1272 does not own an audit key and explicitly forbids self-certification;
- historical #1084 owns only rejected predecessor head ab457e641d3ca5252cdbbd7fd5e2928d3c761521 and its lease is expired/released from current repaired-head authority;
- therefore #1275 wins current-head read-only independent qualification and no Product ownership is taken.

ROUTING / CANDIDATE COMPARISON:
1. PR #1100 repaired exact head 1581dc99... — SELECTED. Producer #1272 released Product ownership and explicitly requires a NEW different-worker audit after closing AUD1272-001/002; exact-head CI is terminal green.
2. Current D03 decontam/G05-G06/balance/split and D04 unique-loss/readiness/recovery/D06/D07/pilot/integration — YIELD to fresh owners/auditors.
3. New source-family acquisition — REJECT as farther from first legal optimizer step.

OWNED_SURFACES:
- read-only exact-head audit of PR #1100 @ 1581dc99a87a38eda777988578ba0a8a9af689d8;
- verify repair delta and durable execution evidence/test semantics;
- independently bind Git execution commit/tree/blob identities, Actions run/job/attempt and artifact metadata/digest;
- verify current-carried config/materializer/focused-test blobs equal execution-time blobs;
- adversarially inspect strict typing/closed-world/tamper rejection and zero-credit truth boundary;
- publish one exact-head verdict and hand it to producer/integrator lineage.

AVOID_SURFACES:
- no mutation of PR #1100 branch/Product files;
- no source rerun merely for audit;
- no source-rights approval or corpus promotion;
- no global dedup/decontam/privacy/balance/split/packing/tokenizer/model/trainer/recovery/evaluation/final-test mutation;
- no merge/integration (#1238 owns integration);
- no paid compute, foreign pretrained weights, external LLM/API data/intelligence.

OBJECTIVE:
Independently prove or reject that repaired PR #1100 makes the historical genuine LOCAL_FREE Ubuntu IRC execution evidence durable and fail-closed: exact execution commit/tree and workflow/config/materializer/focused-test Git blobs, exact run/job/attempt/runner/artifact metadata and digest, exact measured source/materialization hashes/counts, strict non-bool numeric typing, closed-world schema, and current Product-carried byte equality must all bind without widening rights/corpus/training authority.

PACKAGE_DIMENSIONS:
- implementation_or_primary_research
- focused_tests
- adversarial_or_negative_tests
- machine_readable_evidence_or_validator
- live_authority_binding
- end_to_end_or_integration_proof
- documentation_or_operator_handoff

ACCEPTANCE:
1. arbitration WIN — SATISFIED;
2. PR remains exact head 1581dc99...;
3. repair delta narrow evidence+test only;
4. execution commit/tree/blobs externally verified;
5. real Actions run/job/artifact metadata externally verified;
6. current-carried config/materializer/focused-test blobs equal execution-time blobs;
7. strict types/closed-world/tamper regressions verified;
8. exact-head shared CI terminal SUCCESS with substantive jobs executed;
9. zero-credit/source-rights/training truth boundary preserved;
10. publish PASS_FOR_INTEGRATION_REAL_EXECUTION_EVIDENCE / CHANGES_REQUIRED / BLOCKED_BY_EXTERNAL only.

FALLBACK:
Head movement invalidates verdict. Scientific defect returns CHANGES_REQUIRED to same PR lineage. Auditor does not merge or mutate Product.

TRUTH_BOUNDARY:
REAL_DATA_EXECUTED_BY_THIS_AUDIT=false
HISTORICAL_REAL_DATA_EXECUTED=true (candidate-only execution)
CANONICAL_CAPACITY_CREDITED=0
FAMILY_CREDIT_ADDED=0
TRAINING_AUTHORIZED_BYTES=0
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
TOKENIZER_FIT_AUTHORIZED=false
OPTIMIZER_UPDATES_EXECUTED=0
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
FINAL_TEST_ACCESSED=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false
EXTERNAL_LLM_OR_API_USED_FOR_DATA_OR_INTELLIGENCE=false

**Comment by Oleksii-debug** [comment]
## TERMINAL VERDICT — PASS_FOR_INTEGRATION_REAL_EXECUTION_EVIDENCE

TARGET_PR: #1100
TARGET_HEAD: `1581dc99a87a38eda777988578ba0a8a9af689d8`
AUDITOR: SWARM-1275
RESULT: **PASS_FOR_INTEGRATION_REAL_EXECUTION_EVIDENCE**
REVIEW_RECORD: PR #1100 review `5179239537` (COMMENT; GitHub account-level APPROVE is mechanically prohibited on own PR)

Acceptance closure:
1. direct OPEN postclaim arbitration exhausted pages 1..9 and won current-head audit ownership — PASS;
2. PR remained open/non-draft on exact head `1581dc99...` at verdict time — PASS;
3. rejected->repaired delta is two commits modifying only durable evidence JSON + terminal regression test — PASS;
4. execution commit/tree/workflow/config/materializer/focused-test blobs externally rebound to Git objects — PASS;
5. Actions run `34549598850`, attempt 1, job `103109520811`, runner/image/Python and artifact `10180292836` metadata/digest externally rebound — PASS;
6. repaired-head carried config/materializer/focused-test Git blobs equal execution-time blobs; temp workflow absent — PASS;
7. closed-world schema, strict exact int/bool typing and adversarial identity/tamper regressions fail closed — PASS;
8. exact-head shared CI `34604427566` SUCCESS with workflow budget + Ruff + full pytest actually executed — PASS;
9. downloaded original artifact ZIP SHA-256 exactly matches Actions digest and its sole text-free receipt matches durable measured facts — PASS;
10. zero-credit / no-training / no-learned-weights / no-final-test scientific boundary preserved — PASS.

Measured execution remains evidence-only: scanned=4096; retained=972; retained_normalized_bytes=4,799,981; candidate_payload_bytes=5,133,301; source SHA256=`75e38b...c1e60`; candidate SHA256=`d6a0ea...eb6c2`; report SHA256=`0fac5a...5253`.

No Product mutation, no source rerun, no merge, no authority widening. #1238 remains integration owner.
